### ISSUE TITLE
Tried it, but didn't work

### ISSUE THREAD

**Issue by brycewray** [gfscott/eleventy-plugin-youtube-embed#8]
Your plugin sounds great, but I couldn't get it to work. My Markdown parser produced the following HTML, so you may immediately see what the problem was (i.e., maybe it's because it's not on the same line as the `<p>` and the `</p>` tags):
```
<p>
    www.youtube.com/watch?v=u8pCsfT1gDU
</p>
```

And I had also tried it with `https:` in front. Either way, it only kept getting changed into just a link without the embed occurring.

**Comment by gfscott** [comment]
Ah, got it. Sounds like we need to accept arbitrary whitespace between the opening and closing tags and the URL string itself. Should have some time to do this in the next couple days.

Just out of curiosity, have you made any modifications to the markdown parser that Eleventh uses out of the box, or are you using a different one?

**Comment by brycewray** [comment]
@gfscott I am using markdown-it. Sorry I didn’t mention that earlier. Thank you, and I look forward to the fix you mentioned!

**Comment by gfscott** [comment]
@brycewray — Can you try upgrading to 1.1.1 and see if it works for you?

**Comment by brycewray** [comment]
@gfscott  will do. Thanks!

**Comment by brycewray** [comment]
@gfscott Unfortunately, same result. Renders as a link, not as an embedded video. 

The Markdown looks like this (with a blank line above it and a blank line below it)...

```

https://www.youtube.com/watch?v=u8pCsfT1gDU

```

... and the resultant parsing looks like this...
```
<p>
    <a href="https://www.youtube.com/watch?v=u8pCsfT1gDU">https://www.youtube.com/watch?v=u8pCsfT1gDU</a>
</p>
```

**Comment by gfscott** [comment]
@brycewray OK, one more try — 1.1.2 adds support for markdown-it’s `linkify` setting, which automatically wraps URLs in anchor tags. I think that should take care of this case. Please let me know! 🤞

**Comment by brycewray** [comment]
@gfscott Will give it a shot tomorrow night. If all else fails, will also try with `linkify` turned off, if possible.

**Comment by brycewray** [comment]
@gfscott Woke up earlier than usual so tried again with 1.1.2. No joy. Then I turned off `linkify` as I'd said I would try --- again, no change. Even turned off `html` just to see if that helped, and all that did (of course) was escape-out some other HTML on the page (which I also deleted briefly as an experiment). Perhaps I'm just an edge case. If it's working for others, don't sweat it. Thanks very much, and I hope at some point I can figure out why my particular setup won't play nicely with this plugin. :clap
